app/consumer.py

```diff
@@ -60,6 +61,7 @@ def on_message(self, unused_channel, basic_deliver, properties, body):
 
 def main():
     logger.debug("Starting consumer")
+    logger.info("Current version: {}".format(__version__))
```

Comment:
Why not a single log line (info level) with the event and the version, e.g.
```
logger.info("Starting consumer", version=__version__)
```
(If you can embed the version like that - you may need the format stuff)